attestation-agent/kbs_protocol/src/client/rcar_client.rs

```diff
@@ -193,8 +193,12 @@ impl KbsClient<Box<dyn EvidenceProvider>> {
         nonce: String,
     ) -> Result<String> {
         debug!("Challenge nonce: {nonce}");
-        let mut hasher = Sha384::new();
-        hasher.update(runtime_data);
+        let mut hasher: Box<dyn DynDigest + Send> = match tee {
+            Tee::AzTdxVtpm => Box::new(Sha256::new()),
+            _ => Box::new(Sha384::new()),
+        };
```

Comment:
hmm, afaik sha384 is hardcoded [in CoCo AS](https://github.com/confidential-containers/trustee/blob/b7defdada4909f70b860285b752b0fe1f5c55e60/attestation-service/src/token/simple.rs#L23). I understand that ITA needs this to be sha256/sha512 in this case, but do we need changes on the protocol level to support both variants?

Reply:
The `rcar_client.rs` changes in this PR will be dropped in the next push. This will be handled as part of https://github.com/confidential-containers/trustee/issues/242 work landing soon.

Reply:
Reverted changes from rcar_client